app/models/product.js

```diff
@@ -0,0 +1,52 @@
+// OKAY SO NOT SURE IF I NEED ANYTHING OTHER THAN getALL
+
+var fs = require('fs');
+
+let products = [];
+
+fs.readFile("/Users/StephB/code/node-js/sunglasses-io/initial-data/products.json", "utf8", (error, data) => {
```

Comment:
The full file path is unnecessary here and ensures that your app won't run on anyone else's machine - keep this in mind in the future.